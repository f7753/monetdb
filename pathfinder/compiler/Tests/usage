#!/bin/sh

Mlog -x $MTIMEOUT pf -h | sed 's|^ \( \-[a-zA-Z][^:]*: \)|=\1|'

