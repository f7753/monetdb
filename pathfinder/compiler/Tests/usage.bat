@echo on
@prompt # $t $g  

	%MTIMEOUT% pf -h

