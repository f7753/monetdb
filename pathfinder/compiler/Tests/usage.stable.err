stderr of test 'usage` in directory 'compiler` itself:


# 16:36:12 >  
# 16:36:12 >  Mtimeout -timeout 300 ./usage usage 
# 16:36:12 >  


# 16:36:12 >  
# 16:36:12 >  Mtimeout -timeout 60 Mserver --config= -?
# 16:36:12 >  


# 15:32:19 >  
# 15:32:19 >  Done.
# 15:32:19 >  

