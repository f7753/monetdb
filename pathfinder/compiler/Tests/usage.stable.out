stdout of test 'usage` in directory 'compiler` itself:


# 15:32:19 >  
# 15:32:19 >  Mtimeout -timeout 120 ./usage usage 
# 15:32:19 >  


# 15:32:19 >  
# 15:32:19 >  Mtimeout -timeout 60 pf -h
# 15:32:19 >  

Pathfinder XQuery Compiler ($Revision$, $Date$)
(c) University of Konstanz, DBIS group

Usage: pf [OPTION] [FILE]

  Reads from standard input if FILE is omitted.

  -h, --help: print this help message
  -q, --quiet: do not print informational messages to log file
  -l file, --log file: specify log file name (default stderr)
  -d port, --daemon port: act as dæmon listening on specified TCP port
  -o monet|xterm|html, --output-type monet|xterm|html: specify output markup
     (default: monet)
  -P, --print-human-readable: print internal tree structure human-readable
  -D, --print-att-dot: print internal tree structure in AT&T dot notation
  -T, --timing: print elapsed times for compiler phases
  -O, --optimize: enable (expensive) optimizations
  -t, --typing: print static types (in {...}) for core (implies -P)
  -p, --stop-after-parsing: stop processing after parsing
  -s, --stop-after-semantics: stop after semantical analysis
  -n, --stop-after-normalizing: stop after core tree normalization
  -c, --stop-after-core-compilation: stop after core language generation
  -M: print out temporary MIL
  -m, --stop-after-mil-generation: stop after MIL code generation

Enjoy.

# 15:32:19 >  
# 15:32:19 >  Done.
# 15:32:19 >  

